src/Socket/chats.ts

```diff
@@ -168,15 +168,19 @@ export const makeChatsSocket = (config: SocketConfig) => {
 
 	const onWhatsApp = async(...jids: string[]) => {
 		const query = { tag: 'contact', attrs: {} }
-		const list = jids.map((jid) => ({
-			tag: 'user',
-			attrs: {},
-			content: [{
-				tag: 'contact',
+		const list = jids.map((jid) => (
+			const content = (!jid.startsWith('+')) ? `+${jid}` : jid;
+			
+			return {
+				tag: 'user',
 				attrs: {},
-				content: jid,
-			}],
-		}))
+				content: [{
+					tag: 'contact',
+					attrs: {},
+					content,
```

Comment:
Well yes that would restore the previous behaviour but if someone (like me) added the "+" externally on the call of `socket.onWhatsApp` that will result in a double "++" and I don't know if it will return exact results, hence I added a check.

Reply:
> Well yes that would restore the previous behaviour but if someone (like me) added the "+" externally on the call of `socket.onWhatsApp` that will result in a double "++" and I don't know if it will return exact results, hence I added a check.

if so, i have no objection